src/aeso_compiler.erl

```diff
@@ -12,36 +12,32 @@
         , file/2
         , from_string/2
         , check_call/4
-        , create_calldata/3  %% deprecated
+        , create_calldata/3
         , create_calldata/4
         , version/0
         , numeric_version/0
-        , sophia_type_to_typerep/1
-        , to_sophia_value/4  %% deprecated, need a backend
+        , to_sophia_value/4
         , to_sophia_value/5
-        , decode_calldata/3  %% deprecated
+        , decode_calldata/3
         , decode_calldata/4
         , parse/2
         , add_include_path/2
         , validate_byte_code/3
         ]).
 
 -include_lib("aebytecode/include/aeb_opcodes.hrl").
--include("aeso_icode.hrl").
 -include("aeso_utils.hrl").
 
 
 -type option() :: pp_sophia_code
                 | pp_ast
                 | pp_types
                 | pp_typed_ast
-                | pp_icode
                 | pp_assembler
                 | pp_bytecode
```

Comment:
The function `pp_bytecode` was removed, do you still need that? There is also a reference for `pp_bytecode` in `docs/aeso_compiler.md` in case you want to remove this.